src/component/term/TermMetadataEdit.tsx

```diff
@@ -99,6 +108,20 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
                                          help={i18n("term.label.help")}/>
                         </Col>
                     </Row>
+                    <Row>
+                        <Col xs={12}>
+                            <StringListEdit list={this.state.altLabels || []}
```

Comment:
Why is the array sanitized here, when it is also sanitized in the `StringListEdit` component? This way, the list is IMHO sanitized unnecessarily multiple times. I suggest leaving the sanitization in the `StringListEdit` component, and not doing it here. Also, we should probably stick to a single method of sanitization - either the `value || []` or `Utils.sanitizeArray` (the second version being able to handle accidental singular value as well).

Reply:
fixed